src/renderer/models/account.ts

```diff
@@ -0,0 +1,136 @@
+import {Keypair} from "@chainsafe/bls/lib/keypair";
+import {readdirSync, readFileSync} from "fs";
+import {ICGKeystore, ICGKeystoreFactory} from "../services/interfaces";
+
+export interface ISSZAccount {
+    name: string;
+    directory: string;
+    sendStats: boolean;
+}
+
+export class CGAccount implements ISSZAccount {
+    public name: string;
+    public directory: string;
+    public sendStats: boolean;
+
+    private unlocked = false;
+    private validators: Keypair[] = [];
+    private keystoreTarget: ICGKeystoreFactory;
+
+    constructor(name: string, directory: string, sendStats = false, keystoreTarget: ICGKeystoreFactory) {
```

Comment:
- might be easier if constructor accepts `account: IAccount` as first param
- you should provide default KeystoreFactory